app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailViewModel.kt

```diff
@@ -36,7 +37,8 @@ class EventDetailViewModel @Inject constructor(
                 val event = repository.getEventById(state.eventId)
 
                 state = state.copy(
-                    event = event
+                    event = event,
```

Comment:
You check the photos.size < 10 couple of times

Have a function for it, or a constant for the amount of photos
Since if we change to 20 photos, you'll have to change it in 2 places